src/utils/cpUtils.ts

```diff
@@ -33,7 +33,7 @@ export namespace cpUtils {
 
             childProc.on('error', reject);
             childProc.on('close', (code: number) => {
-                if (code !== 0) {
+                if (code !== 0 && command !== 'npm') {
```

Comment:
Let's keep cpUtils agnostic of the command (aka I don't like having 'npm' specific logic in here)